1.10/tutorials/dcos-101/app1.md

```diff
@@ -3,12 +3,14 @@ post_title: Deploying First Application
 menu_order: 3
 ---
 
+Welcome to part 3 of the DC/OS 101 Tutorial
+
+<table class="table" bgcolor="#FAFAFA"> <tr> <td align=justify style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;">**Important:** Mesosphere does not support this tutorial, associated scripts, or commands, which are provided without warranty of any kind. The purpose of this tutorial is to demonstrate capabilities, and may not be suited for use in a production environment. Before using a similar solution in your environment, you must adapt, validate, and test.</td> </tr> </table>
```

Comment:
Global suggestion: "The purpose of this tutorial is to demonstrate capabilities. It may not be suited for use in a production environment."

Otherwise, we're saying the _purpose_ of the tutorial may not be suited to production.